src/Microsoft.AspNetCore.Session/DistributedSession.cs

```diff
@@ -65,6 +71,33 @@ public class DistributedSession : ISession
             _isNewSessionKey = isNewSessionKey;
         }
 
+        public string Id
+        {
+            get
+            {
+                Load(); // TODO: Silent failure
```

Comment:
Because it's an opaque value primary useful for logging. We do the same for request id, connection id, etc..